internal/generator/kzg/template/kzg.go.tmpl

```diff
@@ -23,12 +23,22 @@ var (
 // Digest commitment of a polynomial.
 type Digest = {{ .CurvePackage }}.G1Affine
 
-// SRS stores the result of the MPC
-type SRS struct {
+// Proving and Verifying keys together constitute the SRS (result of the MPC)
+type ProvingKey struct {
```

Comment:
godoc format: struct doc should be // ProvingKey ...

(else it isn't render properly in godoc)